apps/omg/lib/omg/ethereum_event_listener/measure.ex

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.EthereumEventListener.Measure do
+  @moduledoc """
+  Counting business metrics sent to DataDog
```

Comment:
Not sure why I'm noticed this or am pointing it out, but Datadog has a lowercase 'd' in dog. I think this comes from working at GitHub and seeing people write it Github all the time. 😆

Reply:
nice! I admit I didn't check!